CRM/CloseAccountingPeriod/BAO/CloseAccountingPeriod.php

```diff
@@ -348,7 +351,7 @@ public static function createFinancialAccountBalance($params) {
    * @param int $contactID
    *
    */
-  public static function getPriorFinancialPeriod($contactID) {
+  public static function getPriorFinancialPeriod($contactID = NULL) {
```

Comment:
In what case $contactID will be null? We are storing prior financial period against contact.